docs/development/decision-records/2024-09-16-proposal-federated-catalog-distribution/README.md

```diff
@@ -0,0 +1,24 @@
+# Proposal for FederatedCatalog with Tractus-X distribution
+
+## Decision
+
+The proposed solution is to have a Federated Catalog as own runnable service capable of crawling all the chosen catalogs and expose that data.
```

Comment:
this should be an unequivocal decision, not a proposal. It is a "decision record", not a "proposal record". words matter.
Also, what is a "runnable service"? We should be very explicit here, for example, in saying: "The Federated Catalog will be deployed as standalone [component](https://github.com/eclipse-edc/eclipse-edc.github.io/tree/main/developer/decision-records/2024-08-06-components). For this, the Helm charts will be updated to feature a new deployment unit called 'federated-catalog'".